pages/_document.js

```diff
@@ -1,22 +1,13 @@
 import Document, { Head, Main, NextScript } from 'next/document'
-import { ServerStyleSheet } from 'styled-components'
-
-import globalStyle from '../components/globalStyle'
-import meta from '../config/meta'
 
 export default class MyDocument extends Document {
   render () {
-    const sheet = new ServerStyleSheet()
-    const main = sheet.collectStyles(<Main />)
-    const styleTags = sheet.getStyleElement()
     return (
       <html>
-        <Head>
-          {styleTags}
-        </Head>
```

Comment:
Since, we are not using `injectGlobal` anymore, that's why had to remove these things.